website/docs/components/rich-tooltip/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,207 @@
+## Usage
+
+### When to use
+
+- For adding extra information to a UI element that consists of more than a single sentence.
+- When it’s necessary to persist extra information in the UI; e.g., if the user benefits from cross-referencing information in the component with other elements in the UI.
+
+### When not to use
+
+- For extra information that consists of basic text, consider using the [Tooltip](/components/tooltip) instead.
+- For complex functions that change or alter the UI, consider a [Dropdown](/components/dropdown) instead.
+- As a double-confirmation for destructive actions, use a [Modal](/components/modal) instead.
+- If the information is **essential** to the user completing a task. Consider placing this information within the page and associated with a specific UI element.
+
+### Rich Tooltip vs. Tooltip
+
+While there is overlap between the Rich Tooltip and the "standard" [Tooltip](/components/tooltip), which component to use depends on:
+
+- the complexity of the content
+- whether structural or layout elements are used
+- whether there are interactive elements; e.g., links or micro-interactions
+- whether the component should persist within the UI
```

Comment:
I've tried to stop myself from using "ephemeral" so much because it's a verbose term. But I've added some content in here stressing the "temporary" nature of persistence.